pkg/k8sutil/k8sutil.go

```diff
@@ -157,3 +160,18 @@ func CreateOrUpdateEndpoints(eclient clientv1.EndpointsInterface, eps *v1.Endpoi
 
 	return nil
 }
+
+// GetMinorVersion returns the minor version as an integer
+func GetMinorVersion(dclient discovery.DiscoveryInterface) (int, error) {
+	v, err := dclient.ServerVersion()
+	if err != nil {
+		return 0, nil
```

Comment:
Why not propagate the error? Maybe we should even crash if we can't find this out.